Dna_Rna.py

```diff
@@ -0,0 +1,28 @@
+class Rna:
```

Comment:
It's possible to create RNA object that contains 'T' nucleotide, so may be we need some classifier of input? I.e. if we create RNA object, the string can contain only A and T and U and C nucleotides.